Add container service events (#36921)
Azure/azure-sdk-for-net: sdk/eventgrid/Azure.Messaging.EventGrid/CHANGELOG.md

# Release History

## 4.17.0 (2023-06-08)

### Features Added

- Added new system events for Container Service. 

### Breaking Changes

> - `EventGridClient` is removed from the beta releases. It will be included in a subsequent release.

Reviewer: ```suggestion
- `EventGridClient` had been removed for the stable release.  It will be included in the next beta release.
```

Author: I'm not sure if we will have another beta for it or not.

Reviewer: Feel free to tune.   The original text read to me like we were taking `EventGridClient` out of a beta release - but this is a stable one.

Author: Ah yeah - you are right. Fixed.
